python/dev/release_default_linux_spark312.sh

```diff
@@ -26,34 +26,40 @@ BIGDL_DIR="$(cd ${RUN_SCRIPT_DIR}/../..; pwd)"
 echo $BIGDL_DIR
 
 if (( $# < 3)); then
-  echo "Usage: release_default_linux_spark312.sh version quick_build upload mvn_parameters"
-  echo "Usage example: bash release_default_linux_spark312.sh default false true"
-  echo "Usage example: bash release_default_linux_spark312.sh 0.14.0.dev1 false false"
-  echo "Usage example: bash release_default_linux_spark312.sh 0.14.0.dev1 false false -Ddata-store-url=.."
+  echo "Usage: release_default_linux_spark312.sh version quick_build upload suffix mvn_parameters"
+  echo "Usage example: bash release_default_linux_spark312.sh default false true true"
+  echo "Usage example: bash release_default_linux_spark312.sh 0.14.0.dev1 false false true"
+  echo "Usage example: bash release_default_linux_spark312.sh 0.14.0.dev1 false false false -Ddata-store-url=.."
   exit -1
 fi
 
 version=$1
 quick=$2
 upload=$3
-profiles=${*:4}
+if (( $# < 4)); then
```

Comment:
This is to make jenkins jobs compatible, so that only need to modify the jobs that have -Ddata-store-url